roles/confluent.ksql/tasks/main.yml

```diff
@@ -112,6 +112,25 @@
     group: "{{ksql_group}}"
   notify: restart ksql
 
+- name: Ksql Jaas Config
```

Comment:
please remove the jaas config task, this template is removed/ no longer needed in 55x ksql